.env-example

```diff
@@ -0,0 +1,17 @@
+MAS_PUBLIC_WEBSITE_URL=http://<frontendhosturl>/:locale/
+MAS_CMS_URL=http://localhost:3000
+MAS_CMS_API_TOKEN=mytoken
+MAS_CONTENT_SERVICE_URL=http://localhost:8080/content-service/
+GOOGLE_API_KEY=xxxxx-xxxx
+GOOGLE_API_CX_EN=xxxx:xxxx
+GOOGLE_API_CX_CY=xxxx:xxxx
+STATSD_HOST=localhost
+STATSD_PORT=8125
+WEBCHAT_DOMAIN=xxxx.xxxx
+
+CAR_COST_TOOL_DOMAIN=localhost
+CAR_COST_TOOL_CAP_SUBSCRIBER_ID=''
+CAR_COST_TOOL_CAP_PASSWORD=''
+CAR_COST_TOOL_MAX_CARS_TO_COMPARE=3
+AUTH_USERNAME=admin
+AUTH_PASSWORD=password
```

Comment:
the MAS_CONTENT_SERVICE_URL can be removed but we need to keep the MAS_PUBLIC_WEBSITE_URL or rename it as the tests on frontend fail locally for me without it